cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicas.java

```diff
@@ -49,6 +49,7 @@
  */
 public class SortedReplicas {
   private final Broker _broker;
+  private final Disk _disk;
```

Comment:
(Based on previous response) But it looks like one of the constructors below sets `_disk` to `null` -- and this is called from capacity goals (transitively). Is it fair to say that disk is not initialized when the rebalance does not care about the per-disk utilization information and deals with only per-broker utilization information?

Reply:
yes, bascially if `_disk` is null, it means this is a `SortedReplicas` for the broker;  otherwise it is a `SortedReplicas` for the disk. 
With that said, currently only the two intra-broker goal will create `SortedReplicas` for disks.

Reply:
Thanks, that makes it more clear -- can we document this implication in JavaDoc of this class? -- i.e. when is `_disk` expected to be `null`?